i18n/rpi-imager_it.ts

```diff
@@ -64,13 +64,13 @@
     </message>
     <message>
         <location filename="../downloadthread.cpp" line="257"/>
-        <source>discarding existing data on drive</source>
-        <translation type="unfinished"></translation>
+        <source>elimina i dati esistenti nell&apos;unità</source>
+        <translation></translation>
     </message>
     <message>
         <location filename="../downloadthread.cpp" line="277"/>
-        <source>zeroing out first and last MB of drive</source>
-        <translation type="unfinished"></translation>
+        <source>azzera il primo e l'ultimo MB dell&apos;unità</source>
+        <translation></translation>
```

Comment:
@bovirus I think you were supposed to edit the `translation` tag, rather then editing the `source` tag?!